test/models/report_test.rb

```diff
@@ -1,7 +1,16 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+
+  def test_report_accepted_by_project_owner_scope
+    report1 = reports(:first_report)
+    report2 = reports(:second_report)
+
+    # Test for report1
+    assert_equal(Report.accepted, [report1])
```

Comment:
this works now. in the future, if new fixtures are added, this test might break, because the scope will find other reports. Usually I test this testing the inclusion of a record in a collection. e.g.:

    assert_includes Report.accepted, report1
    assert_not_includes Report.accepted, report2